kernel/classes/datatypes/ezxmltext/ezxmltexttype.php

```diff
@@ -843,6 +853,8 @@ function batchInitializeObjectAttributeData( $classAttribute )
         $xmlText = "'" . $db->escapeString( $xmlText ) . "'";
         return array( 'data_text' => $xmlText );
     }
+
+    protected $deletedStoredObjectAttribute;
```

Comment:
phpdoc: *List of fully deleted object attributes by id, used to know when we don't need to perform additional url cleanup*